readme.md

```diff
@@ -79,42 +79,26 @@ Likely supports following social networks:
 
 If you need several Likely widgets on the page, just create another `div` with the class `likely` and list the social networks in it.
 
-### Usage with webpack
+### Using as commonjs module
 
-First, install webpack’s [`json-loader`](https://github.com/webpack/json-loader) and add it to your `webpack.config.js`:
+Likely supports commonjs modules, so you can use it within webpack or browserify build systems.
```

Comment:
“Likely supports CommonJS modules” sounds like “Likely can import a CommonJS module” or “Likely can use something that is written in CommonJS” to me. Probably, “Likely can be used as a CommonJS module” is better?